assets/css/sensei-course-theme/buttons.scss

```diff
@@ -1,42 +1,46 @@
-
 .sensei-course-theme {
-
 	&__button {
 		display: inline-block;
 		font-size: 14px;
 
 		&.is-primary,
 		&.is-secondary {
 			padding: 8px 11px;
-			border: solid 1px var(--sensei-course-theme-primary-color);
+			border: solid 1px var(--primary-color);
```

Comment:
The `--sensei-course-theme-primary-color` variable is available outside of the course frame, and should pick up the customized color directly. (It just has a black default now I believe)

Reply:
Ah, probably that wasn't working for me because I had never set the color in the customizer. After setting it, it worked.
I noticed that we could add a default argument to the `get_theme_mod`, but it doesn't make sense to the `default` from `add_setting` not work. Do you have any idea? Otherwise, I can explore it deeper to see what's happening there.

Also, about the variables, I thought you have thought of that variables like `--primary-color` to use for that scope. So should we remove that variables and just use the `--sensei-...` directly?

Reply:
My thinking was this way the components don't have to worry about where this color is coming from, it's just set once at the top, and they can use a shorter scoped version. The full one is only needed for the buttons  because they also show up in the content area. I imagine we'll refine it a bit once we have content styles.

Reply:
On the default color, I left it out of `get_theme_mod` for now. I'm not sure if we want Sensei green, or take a stab at picking up theme primary color there.

Reply:
> The full one is only needed for the buttons because they also show up in the content area

It's one thing I was thinking of. Probably it would be interesting to move the variables a level up to be used also in the content area. Since now we'll use our own theme, it should probably also affect the content there.

> On the default color, I left it out of get_theme_mod for now. I'm not sure if we want Sensei green, or take a stab at picking up theme primary color there.

Not sure if I got what you meant here. But what I meant is that it wasn't working for any color now. The variable is empty until we set that for the first time.